### DISCUSSION TITLE
Journal Entry: Expired project is still shown in the project drop down menu.

### DISCUSSION THREAD

**Discussion by HarryA82** [ledgersmb/LedgerSMB#5867] [Design]
I added a project, with end date in the past. While the project does not shown in other module (i.e. AR - Add Transaction module), it does show up in the General Journal - Journal Entry module.

**Comment by ylavoie** [comment]
The transaction date is set to undef when calling all_business_units or isn't taken into account on a new journal entry. This allows closed project to show up in the selection of business units. The problem is likely to affect all closed business units.

**Comment by ehuelsmann** [comment]
We Need A general design on how to deal with expired values: they need to show on existing transactions, but not be available for selection on new ones.
